changelog/seqera-cloud/v25.1.0-cycle13_1b294ec.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Seqera Cloud v25.1.0_cycle13
+date: 2025-05-22
+tags: [seqera cloud]
+---
+
+## Version bump
+
+- Bumped Nextflow launcher to `25.04.x`.
+
+## Bug fixes
+
+- Fixed an issue in Firefox, related to expired access tokens, where logging out resulted in an infinite loop in the browser.
```

Comment:
not really related to expired access tokens: I would remove the wording and leave:
```Fixed an issue in Firefox where login resulted in an infinite loop in the browser.```